en/docs/SetupAndInstall/DeployingWSO2APIManager/configure_apim_analytics/configure_worker/active_active.md

```diff
@@ -0,0 +1,135 @@
+# Configure a Analytics workers as active-active
+
+To configure API-M Analytics as a scalable deployment, you can use the Active-Active deployment pattern. For an 
+overview of the Active-Active deployment pattern and instructions to configure it, see the following topics.
+
+- [Overview](#overview)
+- [Configuring an active-active cluster](#configuring-an-active-active-cluster)
+
+
+## Overview
+![](../../../../assets/img/setup-and-install/configure_apim_analytics/Active-Active_Deployment.png)
+
+The above diagram represents a deployment where you are not limited to two nodes. You can scale the event processing 
+horizontally by adding more API-M Analytics nodes to the deployment. In this deployment, it is recommended to 
+configure the gateway node to publish events to multiple API-M Analytics nodes in a Round Robin manner to ensure 
+better fault tolerance. The publishing of events can be carried out by one or more APIM gateway nodes.
+
+In order to perform aggregations in a distributed manner and achieve the scalability, this setup uses distributed aggregations.
+
+Distributed aggregations partially process aggregations in different nodes. This allows you to assign one node to 
+process only a part of an aggregation (regional scaling, etc.). In order to do this all the aggregations must have a 
+physical database and must be linked to the same database.
+
+Partitioning aggregations can be enabled at aggregation level and also at a global level. To enable it at the global 
+level, add the following section with the *PartitionById* annotation set to true in the 
+<APIM-ANALYTICS_WORKER_HOME>/conf/worker/deployment.yaml file.
```

Comment:
File names and paths should be within ` `
`<APIM-ANALYTICS_WORKER_HOME>/conf/worker/deployment.yaml`

Reply:
Fixed